.prettierrc.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   endOfLine: 'lf',
   arrowParens: 'avoid',
   tabWidth: 2,
-};
+  semi: false,
```

Comment:
Interesting 😄 I liked having semicolun to make statemenet end explicit but I think it also helps to keep git history clean. No strong preference as long as we're consistent.

Reply:
Haven't got punished for the lack of semicolon the past few years ^^. Think we are okay with this.